transit_odp/otc/models.py

```diff
@@ -116,3 +119,9 @@ def __str__(self) -> str:
     class Meta:
         verbose_name_plural = "Local Authorities"
         verbose_name = "Local Authority"
+
+
+class InactiveService(models.Model):
+    registration_number = models.CharField(max_length=20, blank=False, null=False)
```

Comment:
registration number should be a unique field in the model as a registration can be picked up multiple times of the effective date is in the range for last 3 days.

Reply:
And one more reason it should be unique is that we should only have one row per registration number even if there are more than one variations returned as a response from the OTC API.